apps/console/src/extensions/components/logs/components/infinite-scroll-container.tsx

```diff
@@ -61,6 +83,8 @@ const InfiniteScrollContainer = (props: InfiniteScrollContainerPropsInterface):
 
     const { t } = useTranslation();
     const [ activeIndex, setActiveIndex ] = useState<number[]>([ -1 ]);
+    const [ view, setView ] = useState<boolean>(false);
```

Comment:
Shall we use more descriptive variable names here?
```suggestion
    const [ isViewModalOpen, setIsViewModalOpen ] = useState<boolean>(false);
```